dashboard/app/views/admin_standards/import_standards.html.haml

```diff
@@ -0,0 +1,57 @@
+:ruby
+  courses = [
```

Comment:
"courses" is our most problematic and overloaded term so I think its important to avoid it unless you are actually referring to an instance of the Course model, here and everywhere below that is not part of a User-facing string.  Better to call these scripts.